pkg/cmd/issue/view/view.go

```diff
@@ -110,11 +108,11 @@ func viewRun(opts *ViewOptions) error {
 	defer opts.IO.StopPager()
 
 	if opts.IO.IsStdoutTTY() {
-		return printHumanIssuePreview(opts.IO, issue)
+		return printHumanIssuePreview(opts, issue)
 	}
 
 	if opts.Comments {
-		fmt.Fprint(opts.IO.Out, prShared.RawCommentList(issue.Comments))
+		fmt.Fprint(opts.IO.Out, prShared.RawCommentList(issue.Comments, api.PullRequestReviews{}))
```

Comment:
When I tried that I got a compiler error. `RawCommentList` and `CommentList` do not take in a pointers to `api.PullRequestReviews`, but I can change that so that `nil` can be passed in to them.

Reply:
Ahh sorry, I misread. I thought this was passing a slice, but it's a struct type. Nevermind!